components/tekton/pipelineruns/create-cluster/create-cluster-pipelinerun.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: create-cluster-pipeline-
+  namespace: vse-cicd-catalog
+spec:
+  serviceAccountName: pipeline-credentials-sa
+  pipelineRef:
+    name: create-cluster
+  # yamllint disable rule:line-length
+  # Git-clone task runs as non-root user
+  # Generally, the default permissions for storage volumes are configured for the root user.
+  # To make the volumes accessible by the non-root user,
+  # you will need to either configure the permissions manually or set the fsGroup field
+  # under PodSecurityContext in your TaskRun or PipelineRun.
+  # yamllint enable rule:line-length
+  podTemplate:
+    securityContext:
+      fsGroup: 65532
```

Comment:
Can you add a comment about what this means :)

Reply:
There is an explanation in the comments above. Would you like also to add a comment about the `fsGroup` number?

Reply:
:facepalm: No its fine I didn't read the above comment.